microk8s-resources/actions/disable.istio.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+echo "Disabling Istio"
+
+skip_opt_in_config "allow-privileged" kubelet
+skip_opt_in_config "allow-privileged" kube-apiserver
```

Comment:
The downside to turning off allow-privileged here is that the user may have turned them on manually, before they installed Istio, in which case we are now inadvertently reverting their configuration change.